dumpling/dumpling.go

```diff
@@ -277,4 +276,9 @@ func (m *Dumpling) detectSQLMode() {
 	}
 	m.logger.Info("found upstream SQL mode", zap.String("SQL mode", sqlMode))
 	m.cfg.LoaderConfig.SQLMode = sqlMode
+	if strings.Contains(sqlMode, "NO_BACKSLASH_ESCAPES") {
```

Comment:
we may also use `HasNoBackslashEscapesMode` in TiDB.

Reply:
If it's Aurora upstream, some SQL mode can't be supported using TiDB parser